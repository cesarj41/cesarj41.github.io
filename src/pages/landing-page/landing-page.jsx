export function LandingPage() {
  return <div>landing page for the portfolio</div>;
}
