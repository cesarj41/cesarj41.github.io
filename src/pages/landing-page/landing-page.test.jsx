import { render } from "@testing-library/preact";
import { LandingPage } from "./landing-page";

describe(LandingPage.name, () => {
  it("Should render PortfolioLandingPage", () => {
    const wrapper = render(<LandingPage />);

    expect(wrapper.container).toMatchSnapshot();
  });
});
