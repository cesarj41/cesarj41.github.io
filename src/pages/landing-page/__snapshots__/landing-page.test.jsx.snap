// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LandingPage > Should render PortfolioLandingPage 1`] = `
<div>
  <div>
    landing page for the portfolio!
  </div>
</div>
`;
